tests/test_opensearch_manager.py

```diff
@@ -36,15 +35,14 @@ class TestOpenSearchManagerClass(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        config = create_config()
         opensearch_host_config = {
-            'host': config.get('DB-Section', 'opensearch-host', fallback='localhost'),
-            'port': config.get('DB-Section', 'opensearch-port', fallback='9200'),
+            'host': 'yc-opensearch',
```

Comment:
@dmytro-kyrychenko All OpenSearch tests are failing because of connection error. `yc-opensearch` is not correct value for tests which are running in Github Actions.
In `test.conf` file you can see that `opensearch-host` should have value `localhost`.
Basically you should revert this line as it most probably cause the harm.

Reply:
About this, I removed this line for the tests, because in this file for tests for opensearch we use real value of running opensearch (yc-opensearch locally), and in all other files, which also use value from config it is set to localhost, which I think is then replaced by mocking
So the config value for this file should be different than for every other test file
I am not sure how to solve this

Reply:
@dmytro-kyrychenko I would still revert this back so we can successfully run the tests. Rather change it locally when you will run the tests.